overlord/servicestate/service_control_test.go

```diff
@@ -108,6 +109,224 @@ func (s *serviceControlSuite) mockTestSnap(c *C) {
 	c.Assert(err, IsNil)
 	err = ioutil.WriteFile(filepath.Join(dirs.GlobalRootDir, "etc/systemd/system/snap.test-snap.foo.service"), nil, 0644)
 	c.Assert(err, IsNil)
+
+	return inf
+}
+
+func verifyControlTasks(c *C, tasks []*state.Task, expectedAction, expectedFlag string, expectedServices ...string) {
```

Comment:
`expectedFlag` is never used as a value, only as a non-empty or empty string. This feels weird from an API stand-point.

Reply:
Fair point, done per your other comments. Thanks